tssc/step_implementers/generate_metadata/__init__.py

```diff
@@ -1,5 +1,24 @@
-"""
-tssc.StepImplementers for the 'generate-metadata' TSSC step.
+"""tssc.StepImplementers for the 'generate-metadata' TSSC step.
+
+Step Configuration
+------------------
+All tssc.StepImplementers for this step should
+accept minimally the following configuration options.
+
+| Parameter       | Description
+|-----------------|------------
+| `TODO`          | TODO
+
+Step Results
+------------
+The combination of tssc.StepImplementers for this step should
+minimally produce the following step results.
+
+| Result Key  | Description
+|-------------|------------
+| `version`   | Constructed semantic version (https://semver.org/).
+| `image-tag` | Constructed semantic version (https://semver.org/) modified \
+                to work as a container image tag.
```

Comment:
@itewk Did you want to place an example here like you did for the previous steps?